docs/overrides/main.html

```diff
@@ -360,5 +360,5 @@
 {% endblock %}
 
 {% block announce %}
-Our new LangChain Academy Course Deep Research with LangGraph is now live! <a href="https://academy.langchain.com/courses/deep-research-with-langgraph/?utm_medium=internal&utm_source=docs&utm_campaign=q3-2025_deep-research-course_co" target="_blank">Enroll for free</a>.
+These docs will be deprecated and removed with the release of LangGraph v1.0 in October 2025. <a href="https://docs.langchain.com/oss/python/langgraph/overview" target="_blank">Visit the v1.0 alpha docs</a>
```

Comment:
```suggestion
These docs will be deprecated and removed with the release of LangGraph v1.0 in October 2025. View the latest documentation on our <a href="https://docs.langchain.com/oss/python/langgraph/overview" target="_blank">new docs site</a>
```

Or something similar maybe? Is an `!` too much? We're excited!!!